profiler/src/Demos/Samples.Computer01/NullThreadNameBugCheck.cs

```diff
@@ -0,0 +1,39 @@
+using System;
```

Comment:
Just incase you didn't know, the file header is missing or not located at the top of the file :trollface: 

![image](https://user-images.githubusercontent.com/18755388/229807082-40265bf4-7560-4858-8e24-b5c38dc03d3c.png)